mk/package-and-sign.sh

```diff
@@ -52,6 +52,8 @@ mkdir_clean()
   rm -rf $1 && mkdir -p $1
 }
 
+mkdir_clean ${OUTPUT_DIR}
```

Comment:
@kc284 This is in the package-and-sign.sh script which is run to test the unsigned package and it copies the installers in a separate output folder (inside the TestXenAdminUnsigned folder), not the one created by the Jenkinsfile. 
I added it because the build started to fail in this phase ("Testing package-and-sign script") follwing Gabor's changes to switch to the msi.